src/mercury-rpc/mercury-rpc-sample.yaml

```diff
@@ -1,42 +1,42 @@
-db:
-  rpc_mongo_db: mercury
-  rpc_mongo_collection: rpc
-  rpc_mongo_servers:
-    - localhost:27017
-  rpc_mongo_replica_set: None
-
-  jobs_mongo_db: mercury
-  jobs_mongo_collection: rpc_jobs
-  jobs_mongo_servers:
-    - localhost:27017
-  jobs_mongo_replica_set: None
-
-  tasks_mongo_db: mercury
-  tasks_mongo_collection: rpc_tasks
-  tasks_mongo_servers:
-    - localhost:27017
-  tasks_mongo_replica_set: None
-
-
-redis:
-  host: localhost
-  port: 6379
-
-inventory:
+rpc:
   inventory_router: tcp://localhost:9000
 
-fontend:
-  service_url: tcp://0.0.0.0:9001
-
-backend:
-  service_url: tcp://0.0.0.0:9002
-
-workers:
-  queue_runners: 10
-
-info:
-  name: be1-local1
-  datacenter: local1
-  address: localhost
-  frontend_port: 9001
-  backend_port: 9002
+  db:
+    name: mercury
+    servers:
+      - localhost:27017
+    replica_name:  None
+    jobs_collection: rpc_jobs
+    tasks_collection: rpc_tasks
+
+  redis:
+    host: localhost
+    port: 6379
+    queue_name: rpc_task_queue
+
+  fontend:
```

Comment:
Q: is that suppose to be `fontend` or `frontend`?

Reply:
yes it is, good catch

Reply:
caught it as a typo in my script to update the configuration for the docker stuff